astropy/cosmology/tests/test_units.py

```diff
@@ -389,17 +389,6 @@ def test_distance(self, kind):
         assert_quantity_allclose(dist.to(cu.redshift, equivalency), z)
 
 
-# FIXME! get "dimensionless_redshift", "with_redshift" to work in this
-# they are not in ``astropy.units.equivalencies``, so the following fails
-@pytest.mark.skipif(not HAS_ASDF, reason="requires ASDF")
```

Comment:
Also please remove HAS_ASDF import above. I don't think that is used anymore.

Reply:
Done